### PR TITLE
v1

### PR DESCRIPTION
**Pull request by KristinaTar**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+
+// import App from './App';
+// import store from './store';
+
+// const Root = () => (
+//   <Provider store={store}>
+//     <BrowserRouter>
+//       <App />
+//     </BrowserRouter>
+//   </Provider>
```
Comment (ArturErmolenko) [inline src/index.tsx:13]: don't forget delete comments

File: src/App.tsx [hunk 2]
```diff
@@ -1,20 +1,82 @@
...
+
+// import { isLoading, getMessage } from './store';
+
+// const App = () => {
+//   const loading = useSelector(isLoading);
+//   const message = useSelector(getMessage) || 'Ready!';
+
+//   return (
+//     <div className="App">
+//       <h1>Redux list of posts</h1>
```
Comment (ArturErmolenko) [inline src/App.tsx:14]: delete comments

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -1,20 +1,59 @@
...
+          </select>
+        </label>
+      </header>
+
+      <main className="App__main">
+        <div className="App__sidebar">
+          {isLoading ? <Loader />
+            : (
+              <PostsList />
+            )}
```
Comment (marina-tilniak) [inline src/App.tsx:47]:
```suggestion
          {isLoading
            ? <Loader />
            : <PostsList />}
```

File: src/App.tsx [hunk 4]
```diff
@@ -1,20 +1,59 @@
...
+        </div>
 
-      <Start />
+        <div className="App__content">
+          {selectedPost
+            && (
+              <PostDetails />
+            )}
```
Comment (marina-tilniak) [inline src/App.tsx:54]:
```suggestion
          {selectedPost  && <PostDetails />}
```

**Review by marina-tilniak — CHANGES_REQUESTED** [review]
Add all files to GitHub

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Did you forget to add store files to git? Add them or ping me in your chat `fe_u21_***`

**Pull request closed without merging**